element_zstack/volume.py

```diff
@@ -216,23 +291,28 @@ class Mask(dj.Part):
         """
 
     def make(self, key):
-        # NOTE: convert seg data to unit8 instead of uint64
-        task_mode, seg_method, output_dir, params = (SegmentationTask * SegmentationParamset & key).fetch1(
+        """Populate the Segmentation with results parsed from analysis outputs."""
+
+        # NOTE: convert seg data to unit16 instead of uint64
```

Comment:
What is this referring to?

Reply:
This is supposed to communicate that the make function here populates the Segmentation and Mask tables with the results of the cellpose segmentation. This docstring will be updated in the next commit to communicate this more clearly.

Reply:
Oh, this is something that is that remains from Chris' initial commits. Segmentation data in uint64 format is accepted by BossDB so it does not need to be converted.

Reply:
The comment has been removed